scripts/bootstrap.sh.in

```diff
@@ -11,6 +11,7 @@ RPM=${RPM:-$(command -v rpm)}
 SYSTEMCTL=${SYSTEMCTL:-$(command -v systemctl)}
 YUM=${YUM:-$(command -v yum)}
 SALT_CALL=${SALT_CALL:-salt-call}
+SALT_MASTER_CALL="salt"
```

Comment:
How is this installed?

Reply:
leftover, the salt binary is only in the container